cellpack/bin/upload.py

```diff
@@ -1,10 +1,21 @@
+import sys
 import fire
+from pathlib import Path
+import logging
+import logging.config
+
 from cellpack.autopack.FirebaseHandler import FirebaseHandler
 from cellpack.autopack.DBRecipeHandler import DBUploader
 
 from cellpack.autopack.interface_objects.database_ids import DATABASE_IDS
 from cellpack.autopack.loaders.recipe_loader import RecipeLoader
 
+###############################################################################
+log_file_path = Path(__file__).parent.parent / "logging.conf"
```

Comment:
Are we using the logger anywhere in this file?

Reply:
Thank you for catching that! I ended up using `sys` instead of `log` and forgot to clean up the logger setup, will remove it now